sunbeam-python/sunbeam/utils.py

```diff
@@ -258,7 +258,7 @@ def get_nameservers(ipv4_only=True) -> List[str]:
         nameservers = list(set(nameservers))
     except FileNotFoundError:
         nameservers = []
-    return nameservers
+        return nameservers[:max_count]
```

Comment:
```suggestion
    return nameservers[:max_count]
```
Return should not be inside the except.

Reply:
Fixed in last force push